devicetypes/smartthings/fibaro-heat-controller.src/fibaro-heat-controller.groovy

```diff
@@ -299,7 +306,8 @@ def switchMode() {
 def sendEventToChild(event) {
 	String childDni = "${device.deviceNetworkId}:2"
 	def child = childDevices.find { it.deviceNetworkId == childDni }
-	child?.sendEvent(event)
+	if (state.isChildOnline)
+		child?.sendEvent(event)
```

Comment:
were you having a problem with this being sent too often?

Reply:
Kind of, but it's more like, after I've received Multilevel Report from endpoint 2, I was sending an event to child, and it was bringing it back online every time, meaning that sending "offline" event haven't done a trick. So this flag is a workaround to store real child status in parent device and prevent sending 'fake' events to additional temperature sensor.